src/main/java/io/securecodebox/persistence/defectdojo/service/DefaultImportScanService.java

```diff
@@ -56,7 +56,7 @@ class DefaultImportScanService implements ImportScanService {
      *
      * @param config not {@code null}
      */
-    DefaultImportScanService(final @NonNull Config config) {
+    public DefaultImportScanService(final @NonNull Config config) {
```

Comment:
Why public?

Reply:
Same reason as why the class needs to be public